core/src/main/resources/schema/app/20200518154332_CB-6236_enable_Ranger_Raz_service.sql

```diff
@@ -0,0 +1,11 @@
+-- // CB-6236 enable Ranger Raz service.
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE cluster ADD COLUMN IF NOT EXISTS ranger_raz_enabled boolean NOT NULL DEFAULT false;
```

Comment:
The default value should be added separately as it will lock the table. See:
https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/resources/schema/app/20200504164000_CB-6897_improve_instance_metadata_with_instance_lifecycle.sql

Reply:
https://www.citusdata.com/blog/2018/02/22/seven-tips-for-dealing-with-postgres-locks/